### PR TITLE
add operation document

### PR DESCRIPTION
**Pull request by zhengwanbo**
add operation document

### CODE REVIEW

File: op-guide/tidb-ops-guide.md [hunk 1]
```diff
@@ -0,0 +1,498 @@
...
+所以日常巡检都是围绕着 Grafana 为中心。
+
+### Grafana 监控指标说明
+
+> 目前 Grafana dashboard 整体分为四个dashboard，node_export，PD，TiDB，TiKV。 内容较多，主要在于全面了解整个集群各个组件	的运行情况。  
+> 针对对于日常运维，我们挑选出一些重要的 metrics 指标放在 overview 页面，方便日常运维人员观察集群组件(PD,TiDB,TiKV)使用状态以及集群使用状态 。  
+> 以下为 overview dashboard 说明。  
+
+####PD运行参数
+
```
Comment (buggithubs) [inline op-guide/tidb-ops-guide.md]: #### PD

File: op-guide/tidb-ops-guide.md [hunk 2]
```diff
@@ -0,0 +1,498 @@
...
+   * [Grafana 监控指标说明](#grafana-监控指标说明)
+* [告警说明](#告警说明)
+   * [alert.rule 告警说明](#alertrule-告警说明)
+   * [告警处理](#告警处理)
+      * [TiDB_query_duration TiDB 慢查询](#tidb_query_duration--tidb-慢查询)
+      * [memery_abnormal](#memery_abnormal)
+      * [TiDB_TiKVclient_region_err](#tidb_tikvclient_region_err)
+      * [Drainer_Disk_space_not_enough](#drainer_disk_space_not_enough)
+      * [Drainer_status_timeout](#drainer_status_timeout)
+* [运维相关信息与运维操作](#运维相关信息与运维操作)
```
Comment (buggithubs) [inline op-guide/tidb-ops-guide.md]: 不用显示第四级菜单吧

File: op-guide/tidb-ops-guide.md [hunk 3]
```diff
@@ -0,0 +1,498 @@
...
+PD	|	Store Status  -- Tombstone store	|	下线成功的 TiKV 节点数量	|	
+PD	|	Current storage usage	|	TiKV 集群存储空间占用率	|	超过 80% 应考虑添加 TiKV 节点
+PD	|	leader balance ratio	|	leader ratio 最大的节点与最小的节点的差	|	均衡状况下一般小于 5%，节点重启时会比较大
+PD	|	region balance ratio	|	region ratio 最大的节点与最小的节点的差	|	均衡状况下一般小于 5%，新增/下线节点时会比较大
+PD	|	number of region	|	region数量分布	|
+PD	|	99% completed-cmds-duration-seconds	|	99% pd-server 请求完成时间	|	小于 5ms
+PD	|	average completed-cmds-duration-seconds	|	pd-server 请求平均完成时间	|	小于 50ms
+
+####TiDB运行参数
+
```
Comment (buggithubs) [inline op-guide/tidb-ops-guide.md]: 格式问题

File: op-guide/tidb-ops-guide.md [hunk 4]
```diff
@@ -0,0 +1,498 @@
...
+服务	|	监控对象	|	说明	|	正常范围
+---	|	---	|	---	|	---
+TiDB	|	connection count	  |	从业务服务器连接到数据库的连接数	|	和业务相关。但是如果连接数发生跳变，需要查明原因。比如突然掉为0，可以检查网络是否中断；如果突然上涨，需要检查业务。
+TiDB	|	handle-requests-duration-seconds	|	请求 PD 获取 TSO 响应时间	|	小于100ms
+TiDB	|	TiDB server QPS	|	集群的请求量	|	这个和业务相关
+TiDB	|	statement count	|	单位时间内不同类型语句执行的数目	|	这个和业务相关
+TiDB	|	Query Duration 99th percentile	|	99% 的 query 时长	|	
+
+####TiKV运行参数
+
```
Comment (buggithubs) [inline op-guide/tidb-ops-guide.md]: 格式问题

**Comment by queenypingcap** [comment]
This is a duplicated PR: https://github.com/pingcap/docs-cn/pull/266. Hence closing it.

**Pull request closed without merging**